Additional non-critical build corrections

joshuaboniface opened the pull request:
1. Remove sha256sum from remaining builds; was removed from others.
2. Correct ownership of moved files in repo dir to root.
3. Ensure Debuntu upload into a per-dist folder or things get wonky.
4. Ensure we continue if non-atomic symlinking fails for some reason.

joshuaboniface committed:
Remove sha256sum from Windows build

joshuaboniface committed:
Correct ownership of moved files

nyanmisaka commented:
https://github.com/search?q=repo%3Ajellyfin%2Fjellyfin-ffmpeg%20sha256sum&type=code

I think these sha256sum need to be removed too?

joshuaboniface committed:
Ensure the debuntu release is included

joshuaboniface committed:
Avoid errors if symlinks fail

These aren't atomic and could overlap, which could get weird, but just
trust the last one will work.

joshuaboniface committed:
Remove additional sha256sums

joshuaboniface committed:
Remove refs to sha256sum files
